Tests/Orso.Arpa.Application.Tests/Orso.Arpa.Application.Tests.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Ical.Net" Version="4.2.0" />
```

Comment:
Bitte das Package immer nur in die Projekte packen, wo es wirklich benötigt wird. Ich spare mir jetzt weitere redundante Kommentare. Gilt natürlich für alle weiteren csproj Files

Reply:
das war mir nicht so klar und hab in der IDE einfach bestätigt für alle statt nachzudenken 🙈